### DISCUSSION TITLE
Frameworks

### DISCUSSION THREAD

**Discussion by iTheMaverick** [community/community#74736] [Programming Help]
### Body

What's the best framework out there for newer web developers?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by luke-beep** [comment]
Frontend Frameworks:

1. React.Ts

2. Express

3. Next

4. Vue

5. Angular

6. Svelte

Backend Frameworks:

1. ASP.NET Core (.NET)

2. Flask (Python)

3. CakePHP (PHP)

4. Ruby on Rails (Ruby)

5. Spring (Java)

6. Fiber (Golang)

7. Laravel (PHP)

Remember, these are just my subjective opinions.

**Comment by gitstua** [comment]
Hi @iTheMaverick, thanks for joining the community discussions!

I will let others answer with their opinions but I can share a couple of things which may help you get started:
- [Web Dev for Beginners](https://github.com/microsoft/Web-Dev-For-Beginners#web-development-for-beginners---a-curriculum) 
- You could look for other projects in our [Trending repositories](https://github.com/trending?since=weekly&spoken_language_code=en)
- You can also explore [GitHub stared content by Topic](https://github.com/topics/javascript-framework)

**Comment by deep100x** [comment]
If you know HTML5 & CSS3 well  then you should go for BOOTSTRAP5 because after giving some time to it after that you will learn TAILWIND CSS as quickly as simple html language, because there is 50% class names are same in both frameworks.

So, you can learn bootstrap in max 10 days then give 10 days to tailwindcss and in between learning classnames  build simple projects......

Thats what i get from my experience of learning html css bootstrap tailwindcss and then after you should go for JAVASCRIPT>...
